src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -182,7 +182,7 @@
      * @since 2.10
      */
     @Parameter
-    private String[] ignoredDependencies = new String[0];
+    private final String[] ignoredDependencies = new String[0];
```

Comment:
What is benefit with final variable, which is used as Mojo parameters? 

Those fields will be injected by maven/plexus so reflections on final fields is more complicated - today it is works ...